test/adapters/net_http_persistent_test.rb

```diff
@@ -10,8 +10,13 @@ def setup
         if defined?(Net::HTTP::Persistent)
           # work around problems with mixed SSL certificates
           # https://github.com/drbrain/net-http-persistent/issues/45
-          http = Net::HTTP::Persistent.new('Faraday')
-          http.ssl_cleanup(4)
+          if Net::HTTP::Persistent::instance_methods.include?(:ssl_cleanup)
```

Comment:
#reconnect_ssl exists back to net-http-persistent v2.4.1, so you probably don't need this `if`